@ornikar/jest-config-react-native-web/package.json

```
@@ -12,12 +12,14 @@
     "access": "public"
   },
   "peerDependencies": {
-    "react-native-web": "^0.17.5"
+    "react-native-web": "^0.17.5",
+    "jest": ">=24.0.0"
```

```suggestion
    "jest": ">=26.6.3"
```

on doit etre en 26 partout

---

fixed